git-config/src/file/git_config.rs

```diff
@@ -202,7 +272,7 @@ impl<'event> GitConfig<'event> {
         }
 
         let paths = paths.iter().map(PathBuf::as_path).collect::<Vec<&Path>>();
```

Comment:
Yes, when seeing this I think the input to `from_paths()` should be `Vec<PathBuf>`.

Reply:
indeed